<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>configuration</class>
 <widget class="QDialog" name="configuration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>464</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>260</y>
     <width>171</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
   </property>
  </widget>
  <widget class="QPushButton" name="funcionamientoButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>171</width>
     <height>151</height>
    </rect>
   </property>
   <property name="text">
    <string>img1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="estadisticasButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>100</y>
     <width>171</width>
     <height>151</height>
    </rect>
   </property>
   <property name="text">
    <string>img2</string>
   </property>
  </widget>
  <widget class="QLabel" name="label1">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>70</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Funcionamiento</string>
   </property>
  </widget>
  <widget class="QLabel" name="label2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>70</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Estadisticas</string>
   </property>
  </widget>
  <widget class="QLabel" name="tituloLabel">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>20</y>
     <width>101</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Configuraci√≥n</string>
   </property>
  </widget>
  <widget class="QPushButton" name="atrasButton">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>280</y>
     <width>96</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Cerrrar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>configuration</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>configuration</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
